django/utils/datastructures.py

```diff
@@ -131,18 +131,23 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
-        """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
-        """
+    def _getlist(self, key, default=None):
+        # Returns mutable list of values for the passed key
+        # Used internally to manipulate list of values
```

Comment:
Missing period is my typo. For the future contributions - internal method docstrings are preferred over comments?